src/test/java/ch/sbb/DiscountCardsTicketBookingPriceVerification.java

```diff
@@ -0,0 +1,79 @@
+package ch.sbb;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static org.junit.Assert.assertEquals;
+
+public class DiscountCardsTicketBookingPriceVerification {
+    WebDriver driver;
+    SbbChHomePage sbbChHomePage;
+    SbbChSearchResultPage sbbChSearchResultPage;
+    SbbChPassengerDetailsPage sbbChPassengerDetailsPage;
+    SbbChTripSummaryPage sbbChTripSummaryPage;
+    SbbChLoginPage sbbChLoginPage;
+
+    @Before
+    public void setup() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver");
+        WebDriver driver = new ChromeDriver();
+        WebDriverWait wait = new WebDriverWait(driver, 60);
+    }
+
+    @Test
+    public void verifyCarWithLowestPriceCanBeParked() {
+        sbbChHomePage.navigateToSbbSite();
+        sbbChHomePage.acceptCookies();
+        sbbChHomePage.setDepartureStation();
+        sbbChHomePage.setArrivalStationStation();
+        sbbChHomePage.setTravelDateToNextDay();
+        sbbChHomePage.clickConnectionSearchButton();
```

Comment:
I believe we discussed the fluent api topic back then, but reiterating to mention this option.
If your eturned the page object(`this`) in your page object methods, you could write this as:
```
sbbChHomePage
        .navigateToSbbSite();
        .acceptCookies();
        .setDepartureStation();
        .setArrivalStationStation();
        .setTravelDateToNextDay();
        .clickConnectionSearchButton();
```
It's a purely stylistic thing, and both are fine. The point is that you have options to choose from.

Reply:
I'll leave it so far as it is since I need to read once again about it so I'd feel more confident in it.